frontend/src/websockets/MessageType.ts

```diff
@@ -0,0 +1,12 @@
+enum MessageType {
+  PING = "PING",
+  JOIN_GAME = "JOIN_GAME",
+  WAITING_ROOM_UPDATE = "WAITING_ROOM_UPDATE"
+}
```

Comment:
What about `ERROR` messages ?

Reply:
We need to think deeper I about error handling, I suggest we do a separate PR with a fully-fledged error handling (i.e. either error page, retry mecanism...).